### PR TITLE
Fix: error switching networks metamask

### PR DESCRIPTION
**Pull request by NewCodeOnDaBlock**
### Fix

* \[x] Added entry to ./changelog
* \[x] Add PR label

### CODE REVIEW

**Review by gamalielhere — APPROVED** [review]

**Review by andrewdahn — APPROVED** [review]

File: src/modules/swap/ModuleSwap.vue [hunk 1]
```diff
@@ -1108,12 +1108,6 @@ export default {
       const findToken = this.actualFromTokens.find(item => {
         if (item.contract === this.defaults.fromToken) return item;
       });
-      if (
```
Comment (gamalielhere) [inline src/modules/swap/ModuleSwap.vue:1111]: why was this removed? this doesn't seem to be related to the issue.

**Review by gamalielhere — CHANGES_REQUESTED** [review]

File: src/modules/swap/ModuleSwap.vue (hunk 1 above)
Comment (NewCodeOnDaBlock) [reply src/modules/swap/ModuleSwap.vue:1111]: I've added it back However, I've pinpointed the issue to be here in this block of code. I've been trying to understand why the return on the 'if' on line 1111 returns mainTokenDetails based on the check. So for now, i've set it to 'findToken' and seems to fix the issue.

File: src/modules/swap/ModuleSwap.vue [hunk 2]
```diff
@@ -1123,7 +1123,7 @@ export default {
         this.defaults.fromToken === MAIN_TOKEN_ADDRESS &&
         new BigNumber(this.balanceInETH).gt(0)
       ) {
-        return this.mainTokenDetails;
+        return findToken;
```
Comment (gamalielhere) [inline src/modules/swap/ModuleSwap.vue:1126]: why is this changed? was this relevant to the fix?

**Review by gamalielhere — CHANGES_REQUESTED** [review]

**Review by gamalielhere — APPROVED** [review]
